tests/unittests/GraphOptzTest.cpp

```diff
@@ -283,6 +283,9 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvFP16) {
   ASSERT_EQ(newCV->getNumUsers(), 1);
   Node *save = newCV->getUsers().begin()->getUser();
   EXPECT_TRUE(llvm::isa<SaveNode>(save));
+
+  optimizedF_ = optimizeFunction(F_);
+  checkNumericalEquivalence();
```

Comment:
You need to provide inputs to the graph. before calling `checkNumericalEquivalence`. See `eliminateNoopTile` for an example.